### PR TITLE
release/v1.7.2

### PR DESCRIPTION
**Pull request by chohongm**

### CODE REVIEW

**Comment by sendbird-sdk-deployment** [comment]
[Creating Ticket] 🔖 Done https://sendbird.atlassian.net/browse/SDKRLSD-1337

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+## [1.7.2] (July 04, 2024)
+### Feat:
+- `FormInput` of `FormMesage` now supports two new `MessageFormItem.style.layout` types: `chip` and `textarea`
+- Added `window.sbWidget` interface to allowing control of the widget state and cache clearing from a non-React environment
+
+### Update:
+- `MessageFeedbackModal` is now being displayed within the widget window
+- `autoOpen` is now being ignored in mobile view
```
Comment (AhyoungRyu) [inline CHANGELOG.md:8]: Can we mention this in the README as well? 

There's `autoOpen` prop in https://github.com/sendbird/chat-ai-widget?tab=readme-ov-file#available-props.

**Review by AhyoungRyu — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (chohongm) [reply CHANGELOG.md:8]: done!

**Pull request merged**